landscape/client/configuration.py

```diff
@@ -871,6 +903,17 @@ def main(args, print=print):
         else:
             sys.exit(EXIT_NOT_REGISTERED)
 
+    if config.actively_registered:
+        currently_registered = actively_registered(config)
+
+        info_text = registration_info_text(config, currently_registered)
+        print(info_text)
+
+        if registration_status:
```

Comment:
this needs to be changed to `if currently_registered`. In the case where `config.is_registered` and `config.registration_sent` are both `False` and `config.actively_registered` is `True`, it is undefined, though the script still prints the status:
```
$ sudo ./scripts/landscape-config --actively-registered

Registered:    False
Config Path:   /home/ubuntu/landscape-client/landscape-client.conf
Data Path      /tmp/landscape/
Traceback (most recent call last):
  File "/home/ubuntu/landscape-client/./scripts/landscape-config", line 16, in <module>
    main(sys.argv[1:])
  File "/home/ubuntu/landscape-client/./landscape/client/configuration.py", line 912, in main
    if registration_status:
UnboundLocalError: local variable 'registration_status' referenced before assignment
```